blockchain/txvalidator.go

```diff
@@ -1214,60 +1305,84 @@ func (b *BlockChain) checkRegisterCRTransaction(txn *Transaction) error {
 		// check code and signature
 		if err := checkStandardSignature(program.Program{
 			Code:      info.Code,
-			Parameter: parameter,
+			Parameter: info.Signature,
```

Comment:
should use:
	buf := new(bytes.Buffer)
	buf.WriteByte(byte(len(info.Signature)))
	buf.Write(info.Signature)
	parameter := buf.Bytes()